tests/test_analysis.py

```diff
@@ -26,7 +26,15 @@ def test_harmonic_analysis(self):
       plot(trange, x)
       plot(trange, y)
       show()
-    self.assertAlmostEqual(numpy.linalg.norm(x-y), 0.0)
+    self.assertAlmostEqual(numpy.linalg.norm(x-y), 0.0, 5)
+
+    def test_error_analysis(self):
```

Comment:
This test isn't actually run, because it's indented to be inside test_harmonic_analysis().

Reply:
Indeed. 4 vs 2 space style here ;-) Fixed.